content/blog/2018/wildcard-https-egress-gateway/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Configure Egress Gateway for HTTPS traffic to wildcarded domains
```

Comment:
This should not be a blog. This should be a task. Also, we really really need an advanced task for all variants of egress.

Reply:
Let me change it to a task. So which ideas do you have for the advanced task? Currently we have, in addition to this PR:

https://istio.io/docs/tasks/traffic-management/egress/
https://istio.io/docs/tasks/traffic-management/egress-tls-origination/
https://istio.io/docs/tasks/traffic-management/egress-gateway/
https://istio.io/blog/2018/egress-https/
https://istio.io/blog/2018/egress-tcp/
https://istio.io/blog/2018/egress-monitoring-access-control/